spec/features/user_edits_review_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+feature "user edits review", %{
+  As a signed in user
+  I want to edit my review
+  So I can update my review if something changes
+} do
+  before(:each) do
+    user = FactoryGirl.create(:user)
+    review = FactoryGirl.create(:review)
+    station = FactoryGirl.create(:station)
+    review = Review.new(description: review.description, 
+    rating: review.rating, 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.